jberet-feature-pack/galleon-feature-pack/wildfly-feature-pack-build.xml

```diff
@@ -0,0 +1,32 @@
+<!--
+  ~ JBoss, Home of Professional Open Source.
+  ~
+  ~ Copyright 2021 Red Hat, Inc., and individual contributors
+  ~ as indicated by the @author tags.
+  ~
+  ~ Licensed under the Apache License, Version 2.0 (the "License");
+  ~ you may not use this file except in compliance with the License.
+  ~ You may obtain a copy of the License at
+  ~
+  ~     http://www.apache.org/licenses/LICENSE-2.0
+  ~
+  ~ Unless required by applicable law or agreed to in writing, software
+  ~ distributed under the License is distributed on an "AS IS" BASIS,
+  ~ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+  ~ See the License for the specific language governing permissions and
+  ~ limitations under the License.
+  -->
+<build xmlns="urn:wildfly:feature-pack-build:3.1" producer="org.jberet:galleon-feature-pack">
+    <transitive>
+        <dependency group-id="org.wildfly" artifact-id="wildfly-ee-galleon-pack" translate-to-fpl="true">
+            <name>org.wildfly:wildfly-ee-galleon-pack</name>
+            <packages inherit="true">
+                <exclude name="org.jberet"/>
+            </packages>
```

Comment:
I'll have to look into this, but there is a `batch-jberet` layer and we may need to exclude that instead.